cpp/arcticdb/version/python_bindings.cpp

```diff
@@ -327,6 +338,24 @@ void register_bindings(py::module &version, py::exception<arcticdb::ArcticExcept
         .def_property_readonly("creation_ts", &DescriptorItem::creation_ts)
         .def_property_readonly("timeseries_descriptor", &DescriptorItem::timeseries_descriptor);
 
+    py::class_<StageResult>(version, "StageResult")
+        .def(py::init([]() { return StageResult({}); }))
+	.def_property_readonly("_staged_segments", [](const StageResult& self) { return self.staged_segments; })
+	.def_property_readonly("_version", [](const StageResult& self) { return self.version; })
```

Comment:
Ah great glad you have the `_version` property 👍 

Don't think either property should be protected FWIW, this is just POD.